.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "🐛 Bug report"
+about: Report a bug or regression
+title: ''
+labels: "\U0001F41B bug, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+    Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.
+
+    Your issue might already exist. If so, add a comment to the existing issue instead of creating a new one. You can find existing issues here:
+    - an existing Github issue: https://github.com/hmrc/platui-support/issues
+-->
+
+## Description of the issue
+<!-- A clear and concise summary of what the bug is. -->
+
+## Steps to reproduce the issue
+<!-- How can we reproduce this issue? If you think it will be helpful, please provide a small code snippet and/or screenshots. -->
+
+## Actual vs expected behaviour
+<!-- What is happening vs what would you expect to happen instead? -->
+
+## Environment (where applicable)
+<!-- Details of your operating system, browser, name and version of the library you’re using may help us to reproduce your issue. -->
+
+- Operating system:
+- Browser:
+- Browser version:
+- Libary Name & Version:
```

Comment:
Small spelling mistake, and maybe useful extra request

```suggestion
- Library Name & Version:
- Team / Service / Repo:
```

Though not sure if Team / Service / Repo is asking for too much information via a public channel, @robert-best can you say?